Readme.md

```diff
@@ -3,7 +3,28 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T854758)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-*Files to look at*:
+
+# Data Grid for Blazor - How to use DevExpress Reporting tools to implement export in a WASM application
+
+This example illustrates how to use DevExpress Reporting tools to export Data Grid's content to different file formats (.pdf/.xlsx/.docx) in a Blazor WebAssembly application.
+
+![Resulting Application](images/application.png)
+
+In this example, the Data Grid uses its [CustomData](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDataGrid-1.CustomData) property to [bind to the external Web API service](./CS/DxDataGridExportingWithReportsClientBlazor/DxDataGridExportingWithReportsClientBlazor/Pages/Index.razor#L26). The Web API service source code is in the [DataSourceWebApi folder](./CS/DataSourceWebApi/DataSourceWebApi). The database name is "Northwind". You can find its script generation file in the [DBBackup folder](./CS/DataSourceWebApi/DataSourceWebApi/DBBackup). Generate this data on your SQL server and change the connection string in [NWINDContext.cs file](./CS/DataSourceWebApi/DataSourceWebApi/Models/NWINDContext.cs) (see the OnConfiguring method). 

```

Comment:
It does not read well when you have several short sentences, especially when there seems to be no connection between sentences: "The source code is here. The name is this. The folder is there" and so on